src/pages/ReimbursementAccount/NonUSD/Country/substeps/Confirmation.tsx

```diff
@@ -80,13 +80,16 @@ function Confirmation({onNext}: SubStepProps) {
                         interactive={false}
                     />
                     <Text style={[styles.ph5, styles.mb3, styles.mutedTextLabel]}>
-                        {`${translate('countryStep.yourBusiness')} ${translate('countryStep.youCanChange')}`}{' '}
-                        <TextLink
-                            style={[styles.label]}
```

Comment:
If we don't use TextLink, it will again cause the misaligned text on IOS

![simulator_screenshot_09004549-4156-47BA-A52F-A491576132C1](https://github.com/user-attachments/assets/4905f416-4078-4c82-9e06-cef52ccedd22)

Reply:
Updated